src/helper/url.js

```diff
@@ -9,12 +12,22 @@ export const hostnameContext = () => {
       return 'https://app.dev.openlaw.io/';
     case 'production_production':
       return 'https://app.openlaw.io/';
-
-    // @todo: staging
-    /*case 'production_staging':
-      return 'https://app.openlaw.io/';*/
-
     default:
       return 'http://localhost:9000/';
   }
 };
+
+export const websiteURL = () => {
+  const deployContext = process.env.GATSBY_OL_CONTEXT;
+  const nodeEnv = process.env.NODE_ENV;
+  const env = `${nodeEnv}_${deployContext}`;
+
+  switch(env) {
+    case 'production_develop':
+      return 'https://develop.dev.openlaw.io/';
+    case 'production_production':
+      return 'https://openlaw.io/';
+    default:
```

Comment:
Do you want to also catch the case of `staging.dev.openlaw.io` here?

Reply:
Does that current exist? what's the path to that?